CHANGELOG.md

```diff
@@ -95,6 +95,9 @@ Jobs resource. Here are the highlights of what's in this release:
 
 ### Changed:
 
+- [Removed the physicaName lookup for the logical metric name](https://github.com/yahoo/fili/pull/55)
+  * Updated `TableUtils::getColumnNames`
```

Comment:
This 2nd line isn't really adding any benefit. Or rather, perhaps the 2 lines can be combined?:

```
   - Removed `physicalName` lookup for metrics in `TableUtils::getColumnNames` to remove spurious warnings
     * Metrics are not mapped like dimensions are
```